index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio Website</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"
+    />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <ul class="menu-bar">
+          <li id="menu-icon">Welcome</li>
+          <li>
+            <div class="menu-icon1"><i class="fas fa-bars"></i></div>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <section>
+      <div class="container">
+        <div class="heading-container">
+          <h1 class="heading">Hey there. I’m Mariane.</h1>
+          <h2 class="heading-2">I'm a software developer</h2>
+        </div>
+        <p class="text">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <ul class="icons">
+          <li><i class="fab fa-github"></i></li>
+          <li><i class="fab fa-linkedin-in"></i></li>
+          <li><i class="fab fa-angellist"></i></li>
+          <li><i class="fab fa-twitter"></i></li>
+          <li><i class="fab fa-medium"></i></li>
+        </ul>
+      </div>
```

Comment:
Kindly add the down button shown here in the Figma design.
![image](https://user-images.githubusercontent.com/24703171/153488617-f24d5be7-d9a2-4d11-8076-4e65fc160d29.png)